tests/files/opds/metadata_wrangler_overdrive.opds

```diff
@@ -1,9 +1,10 @@
-<feed xmlns:simplified="http://librarysimplified.org/terms/" xmlns:app="http://www.w3.org/2007/app" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:opds="http://opds-spec.org/2010/catalog" xmlns:schema="http://schema.org/" xmlns="http://www.w3.org/2005/Atom" xmlns:bibframe="http://bibframe.org/vocab/">
+<feed xmlns:simplified="http://librarysimplified.org/terms/" xmlns:app="http://www.w3.org/2007/app" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:opds="http://opds-spec.org/2010/catalog" xmlns:schema="http://schema.org/" xmlns="http://www.w3.org/2005/Atom" xmlns:bibframe="http://bibframe.org/vocab/" xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">
   <id>http://localhost:5000/</id>
   <title>Open-Access Content</title>
   <updated>2015-01-02T16:56:40Z</updated>
   <link href="http://localhost:5000/"/>
   <link href="http://localhost:5000/" rel="self"/>
+  <opensearch:totalResults>201</opensearch:totalResults>
```

Comment:
This verifies that our side of the code will work once https://jira.nypl.org/browse/SIMPLY-2422 is implemented on the metadata wrangler side.